target/linux/mvebu/config-6.12

```diff
@@ -88,7 +88,8 @@ CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_HAS_ASID=y
 CONFIG_CPU_IDLE=y
-CONFIG_CPU_IDLE_GOV_LADDER=y
+CONFIG_CPU_IDLE_GOV_MENU=y
+CONFIG_CPU_LITTLE_ENDIAN=y
```

Comment:
Little endian is not mentioned in the commit message

Reply:
Oops. It was automatically selected when I ran `make kernel_menuconfig`. Will remove that.